wgsl/Makefile

```diff
@@ -1,17 +1,15 @@
+.PHONY: all clean
+
 all: index.html grammar/grammar.js
 
+clean:
+	rm -f index.html index.pre.html grammar/grammar.js
 
-GHC=https://github.com/gpuweb/gpuweb/blob
-index.html: index.pre.html Makefile
-	# Append another line to the "This version:" metadata at the top.
-	sed -e "s,gpuweb/wgsl/</a>,gpuweb/wgsl/</a><br><a href=\"$(GHC)/$(shell git rev-parse HEAD)/wgsl/index.bs\">$(GHC)/$(shell git rev-parse HEAD)/wgsl/index.bs</a>," <index.pre.html >index.html
+index.pre.html: index.bs
+	DIE_ON=everything ../tools/invoke-bikeshed.sh index.pre.html index.bs
 
-index.pre.html: index.bs Makefile
-	bikeshed --die-on=everything spec index.bs index.pre.html
+index.html: index.pre.html
+	../tools/add-git-head-to-bikeshed-header.sh index.pre.html > index.html
 
 grammar/grammar.js: index.bs extract-grammar.py
-	python3 extract-grammar.py index.bs grammar/grammar.js
-
-online:
-	curl https://api.csswg.org/bikeshed/ -F file=@index.bs -F output=err
-	curl https://api.csswg.org/bikeshed/ -F file=@index.bs -F force=1 > index.html
+	./extract-grammar.py index.bs grammar/grammar.js
```

Comment:
Why is python3 prefix lost here? At least in my environment, Python files do not execute as such.

Reply:
Mainly for consistency with the way I'm invoking other things (.sh files). I marked them all as executable and checked the `#!` (shebang) directives so I thought they would work. I can change both to call `python3` and `bash` directly since I can see how that would be more compatible.

(Also, this avoids having duplication between `python3` here and `/usr/bin/env python3` in the shebang. But that's not very important.)